### PR TITLE
Assignment5: Error handling using try catch block.

### PR DESCRIPTION
**Pull request by praveen24896**
error

- You have to think about given image and come with scenarios.
- What are the try and catches are occurring into that image. You have to find different kind of exceptions. You have to write them in code with proper sequence.

### CODE REVIEW

File: Assignment5/CarAccidentScenario.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+    private static void contactFamily() {
+        System.out.println("Contact the injured family members.");
+    }
+
+    private static void contactAuthorities() {
+        System.out.println("Contact the nearby authorities");
+    }
+}
+
```
Comment (PrashantItt) [inline Assignment5/CarAccidentScenario.java]: Breaks file  into single Exception class

**Comment by pritishajain05** [comment]
I think you can use a single try block and throw different exceptions.

File: Assignment5/CarAccidentScenario.java
Comment (pritishajain05) [inline Assignment5/CarAccidentScenario.java:1]: I think you can use a single try block and throw different exceptions.

**Review by PrashantItt — APPROVED** [review]

File: Assignment5/CarAccidentScenario.java
Comment (praveen24896) [reply Assignment5/CarAccidentScenario.java:1]: Modified the code

**Review by puneet-tanwar — APPROVED** [review]

**Review by pritishajain — APPROVED** [review]

**Pull request merged**